### DISCUSSION TITLE
What is the Red Order?

### DISCUSSION THREAD

**Discussion by Garfdsggd** [space-wizards/space-station-14#25018] [Questions]
What is the "Red Order"?Recently in the game, after spawning as an admin, I found ID the card to which It says red order, but I haven’t found any information about it anywhere, maybe you can help?

**Comment by Errant-4** [comment]
Pretty sure that this item was introduced by whatever fork you were playing on. It does not exist in our codebase
Or if it was on Wizard's Den, then an admin must have spawned and named it manually

**Comment by Boaz1111** [comment]
what server was this on?
